commandapi-platforms/commandapi-bukkit/commandapi-bukkit-nms/commandapi-bukkit-1.15/src/main/java/dev/jorel/commandapi/nms/NMS_1_15.java

```diff
@@ -974,4 +976,17 @@ public <T> T getMinecraftServer() {
 		}
 	}
 
+	@Override
+	public void syncCommands() {
+		// For some reason, 1.15 is being stupid and has a private syncCommands method
+		// Reflection it is!
+		try {
+			Method syncCommands = Class.forName("org.bukkit.craftbukkit.v1_15_R1.CraftServer").getDeclaredMethod("syncCommands");
```

Comment:
VarHandles please, safe reflection annotation needed as well.

Reply:
How would that look? Do VarHandles work with methods?